### ISSUE TITLE
ERROR 500 ON WINDOWS

### ISSUE THREAD

**Issue by renatoSancho** [3liz/lizmap-web-client#1078]
Hello everyone,
I followed the steps in the Lizmap docs to install it on windows. But, at the end, when i try to connect as admin, i get this error "Error 500. A technical error has occured. Sorry for this trouble"

**Comment by pcav** [comment]
Please make sure all the permissions are OK, by running the `set_rights.sh` command.

**Comment by renatoSancho** [comment]
set_rights.sh give me this error
"Chown : invalid user : 'www-data:www-data'"

**Comment by gioman** [comment]
> Please make sure all the permissions are OK, by running the `set_rights.sh` command.

not necessary on Windows.

**Comment by jaitor1** [comment]
Change .htaccess content from "Allow from all" to "Require all granted".

See: https://github.com/3liz/lizmap-web-client/issues/1478

**Comment by laurentj** [comment]
@jaitor1 no, this is not related. this error is displayed by lizmap, not by Apache.

@renatoSancho You have something that is wrong. See files into lizmap/var/log/: error.log or errors.log. You should have some errors. If you understand them, probably you should be able to fix your installation. Else report them to us.
